src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/service/UserRolesService.kt

```diff
@@ -0,0 +1,27 @@
+package uk.gov.justice.digital.hmpps.manageusersapi.service
+
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.manageusersapi.resource.Role
+import uk.gov.justice.digital.hmpps.manageusersapi.resource.RoleDetail
+import uk.gov.justice.digital.hmpps.manageusersapi.resource.UserRoleDetail
+
+@Service
+class UserRolesService(
+  val authService: AuthService,
+  val nomisApiService: NomisApiService,
+) {
+
+  @Throws(UserNotFoundException::class)
```

Comment:
do you need this annotation? I thought it was needed when integrating with non-Kotlin languages